.azure-pipelines/client.test.live.yml

```diff
@@ -31,6 +31,7 @@ steps:
       command: 'custom'
       customCommand: 'run live-test-client -- -- -- --reporter mocha-junit-reporter'
     displayName: 'npm run live-test-client'
+    timeoutInMinutes: 90
```

Comment:
I would increase this to 120 minutes.  For these type of timeouts, longer is usually better, so I default to double the expected runtime.  If the tests do run extra-slow for some reason, we'd rather have them pas at say 100 minutes than timeout at 90 minutes.  The timeout is mainly to avoid blocking an agent indefinitely if tests hang.

Reply:
That's a good point. Let's see what the results are with this timeout and then we can increase it as needed.

Reply:
If this build passes I'd say increase to 120 and merge.

Reply:
It looks like it still failed and it looks like it only took 1 hour before it failed -- maybe I am reading the logs wrong.

```
2019-02-01T18:41:07.9190637Z [command]/opt/hostedtoolcache/node/10.15.1/x64/bin/npm run live-test-client -- -- -- --reporter mocha-junit-reporter
2019-02-01T19:39:44.9706007Z ##[error]The operation was canceled.
```